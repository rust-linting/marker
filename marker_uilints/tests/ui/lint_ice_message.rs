//@rustc-env:RUST_BACKTRACE=0
//@normalize-stderr-test: "lib.rs:.*" -> "lib.rs"

// This function will trigger a panic in the `uilints` lint crate.
// I want to see how the ICE message looks and make sure that it
// links to the rust-marker/marker repo.
//
// Note: Panicking across crate bounds is unsound, that's a problem
// we'll be ignoring for now rust-marker/marker#10
fn uilints_please_ice_on_this() {}

fn main() {}
