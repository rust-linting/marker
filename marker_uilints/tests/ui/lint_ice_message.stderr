thread '<unnamed>' panicked at marker_uilints/src/lib.rs
free ice cream for everyone!!!
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
fatal runtime error: Rust cannot catch foreign exceptions
